core/src/main/scala/com/clairvoyant/restonomer/core/config/RestonomerConfigurationsLoader.scala

```diff
@@ -28,7 +28,8 @@ object RestonomerConfigurationsLoader {
 
   def loadConfigsFromDirectory[C: ClassTag](
       configDirectoryPath: String,
-      configVariablesSubstitutor: ConfigVariablesSubstitutor = ConfigVariablesSubstitutor()
+      configVariablesSubstitutor: ConfigVariablesSubstitutor = ConfigVariablesSubstitutor(),
+      fileName: Option[String] = None
```

Comment:
We don't need this fileName parameter anymore.

Reply:
resolved